README.md

```diff
@@ -110,8 +110,26 @@ Stripe::Charge.retrieve(
     api_key: "sk_test_...",
   }
 )
+
+Stripe::Charge.capture(
+  "ch_18atAXCdGbJFKhCuBAa4532Z",
+  {},
+  {
+    stripe_version: "2018-02-28",
+    api_key: "sk_test_...",
+  }
+)
 ```
 
+Keep in mind that there are different method signatures depending on the action:
+ - When operating on a collection (e.g. `.list`, `.create`) the method signature is
+  `method(params, opts)`.
+ - When operating on resource (e.g. `.capture`, `.update`) the method signature is
+  `method(id, params, opts)`.
```

Comment:
Just for cleanliness, can we keep consistent indentation for each bullet? (First two indent at two spaces and the last one indents at three.)

Reply:
Whoops, thanks for catching that!